### DISCUSSION TITLE
Ollama chatbot

### DISCUSSION THREAD

**Discussion by AlessioGesuelliUnicam** [FlowiseAI/Flowise#1795] [Q&A] [answered]
I try to connect Ollama in flowise but return error 404, but if i pass the url on postman it works
<img width="2294" alt="Screenshot 2024-02-22 alle 16 27 19" src="https://github.com/FlowiseAI/Flowise/assets/105433120/afb8bcaa-0c67-4aeb-895e-396eddca5869">
<img width="1045" alt="Screenshot 2024-02-22 alle 16 28 02" src="https://github.com/FlowiseAI/Flowise/assets/105433120/b08a3e05-eded-4a3c-922d-e05a77259dcc">

**Comment by mehdi-belkhayat** [comment]
Hi it works well on my side, but with http://ipaddresse:11434  only ...nothing more than this

**Comment by lde-net2** [comment, marked as answer]
if you run flowise under docker,  it can access ollama   though the url http://host.docker.internal:11434

http://localhost inside docker points to the container itself, not your host machine
https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach
